RELEASE.md

```diff
@@ -0,0 +1,57 @@
+# lnc-web Release Process
+
+This document describes the steps needed to release a new version of
+`@lightninglabs/lnc-web` and publish the package to the NPM registry.
+
+We typically release a new version of `lnc-web` whenever a new version of
+`lnc-core` has been released. We want to keep the version numbers in sync
+for both packages.
+
+The steps below to should be done in a PR with proper review.
+
+## Update WASM binary CDN URL
+
+In [lnc.ts](https://github.com/lightninglabs/lnc-web/blob/main/lib/lnc.ts),
+update the default `wasmClientCode` property to point to the latest version
+of the WASM binary which should have been published to the
+https://lightning.engineering website.
```

Comment:
How does the WASM binary get built and deployed to the lightning.engineering website?

Reply:
That deployment happens in the private repo for the website. I didn't want to mention specifics since this repo is public.